### DISCUSSION TITLE
Can I running gauge with jdk8?

### DISCUSSION THREAD

**Discussion by MarsOu1998** [getgauge/gauge#2014] [General]
![image](https://user-images.githubusercontent.com/46570613/120961990-6f836700-c791-11eb-9102-1f9e437ff845.png)

I am confused,I try to run gauge with jdk8 and it works well.

**Comment by zabil** [comment]
Unfortunately the latest version of Gauge does not support JDK8

**Reply by MarsOu1998** [reply]
Thanks,but why jdk8 is still used in the document?
![image](https://user-images.githubusercontent.com/46570613/121281677-df692d00-c90a-11eb-8373-68377a0c9f53.png)

**Reply by zabil** [reply]
Thanks for pointing that out it is outdated. Do feel free to send a pull request for fixing that reference.

**Reply by MarsOu1998** [reply]
> Thanks for pointing that out it is outdated. Do feel free to send a pull request for fixing that reference.

Ok，I will  send a pull request later
